pliers/extractors/models.py

```diff
@@ -84,3 +84,65 @@ def _extract(self, stim):
             values.append(float(extraction[1]))
 
         return ExtractorResult([values], stim, self, features=features)
+
+
+def _resize_image(image, shape):
+    import numpy as np
+    from PIL import Image
+    return np.array(
+        Image.fromarray(image).resize(shape, resample=Image.BILINEAR))
+
+
+class TensorFlowKerasInceptionV3Extractor(ImageExtractor):
+
+    ''' Labels objects in images using a pretrained Inception V3 architecture
+     implemented in TensorFlow / Keras.
+
+    Args:
+        weights (str): URL to download pre-trained weights. If None (default),
+            uses the pre-trained Inception V3 model (dated 2017-03-10) used in
+            Keras Applications.
+        num_predictions (int): Number of top predicted labels to retain for
+            each image.
+     '''
+
+    _log_attributes = ('weights', 'num_predictions')
+    VERSION = '1.0'
+
+    def __init__(self, weights=None, num_predictions=5):
+        super(TensorFlowKerasInceptionV3Extractor, self).__init__()
+        if weights is None:
+            weights = 'imagenet'
+        self.weights = weights
+        self.num_predictions = num_predictions
+
+        import tensorflow as tf
```

Comment:
We have a convention for importing optional dependencies that allows us to detect missing libraries; see the `attempt_to_import` and `verify_dependencies` utilities used in other extractors.